improvement: Expose hwmon collector config options in prometheus.exporter.unix component. (#1369)
grafana/alloy: docs/sources/reference/components/prometheus/prometheus.exporter.unix.md

```
{{< /code >}}

### hwmon block

| Name               | Type       | Description                                                         | Default  | Required |
|--------------------|------------|---------------------------------------------------------------------|----------|----------|

> | `chip_include`     | `string`   | Regexp of hwmon chip to include mutually exclusive to chip-exclude. |          | no       |

Reviewer: ```suggestion
| `chip_include`     | `string`   | Regexp of hwmon chip to include (mutually exclusive to chip-exclude). |          | no       |
```

Reviewer: If you accept the suggestions in github directly, make sure that you reformat the tables. Same for the comment below

Author: fixed

> | `chip_exclude`     | `string`   | Regexp of hwmon chip to include mutually exclusive to chip-include. |          | no       |

Reviewer: ```suggestion
| `chip_exclude`     | `string`   | Regexp of hwmon chip to exclude (mutually exclusive to chip-include). |          | no       |
```

> | `chip_exclude`     | `string`   | Regexp of hwmon chip to (include mutually exclusive to chip-include). |          | no       |

Reviewer 2: These descriptions are.... hard to understand. 

1. We try to avoid using brackets in the text descriptions.
2. If you do use brackets like this, it's for additional information and the text should make sense without the bracketed additional info.  If I take out the bracketed info I'm left with `Regexp of hwmon chip to` which is incomplete.

Sometimes people use brackets for info that would fit just fine within the sentence. Even taking out the brackets, I'm unable to really understand the description.

Reviewer: The doc here is not correct, the correct version should be:
- Regexp of hwmon chip to include (mutually exclusive to chip-exclude).
- Regexp of hwmon chip to exclude (mutually exclusive to chip-include).

If we want to avoid the brackets, I would suggest the following:
- Regexp of hwmon chip to include
- Regexp of hwmon chip to exclude

And below we can add a text saying "`chip_include` and `chip_exclude` are mutually exclusive".

@clayton-cornell feel free to change the wording

Reviewer 2: Now it makes a LOT more sense. Thanks.

Author: I poorly copied the [upstream](https://github.com/prometheus/node_exporter/blob/f1e0e8360aa60b6cb5e5cc1560bed348fc2c1895/collector/hwmon_linux.go#L35-L36) help docs. Updated to use the promosed new wording.

Reviewer 2: Looks good now.

> | `chip_include`     | `string`   | Regexp of hwmon chip to (include mutually exclusive to chip-exclude). |          | no       |
> | `chip_exclude`     | `string`   | Regexp of hwmon chip to (include mutually exclusive to chip-include). |          | no       |

Reviewer 2: ```suggestion
| `chip_include`     | `string`   | Regular expression of hwmon chip to include. Mutually exclusive to `chip-exclude`. |          | no       |
| `chip_exclude`     | `string`   | Regular expression of hwmon chip to exclude. Mutually exclusive to `chip-include`. |          | no       |
```
